docs/docs/components/service-mesh.md

```diff
@@ -14,6 +14,20 @@ all workloads which have a specified configuration. The service mesh container f
 sets up `iptables` rules based on its configuration and then starts
 [Envoy](https://www.envoyproxy.io/) for TLS origination and termination.
 
+## Service mesh startup enforcement
+Since Contrast doesn't yet enforce the order in which the containers are started
+(see [Limitations](../features-limitations.md)), we deny all incoming connections
+until the service mesh resolves this deny rule.
+A systemd unit inside the podVM creates this deny rule.
+The kata-agent systemd unit requires that this unit successfully runs and exists.
+Therefore, the deny rule is in place, before any containers can be started.
+
+Contrast ensures that this deny rule doesn't drop traffic in scenarios that don't
+require a service mesh by injecting the service mesh container not as a sidecar
```

Comment:
This should be more connected to action of a user: "by setting flag -xy during generate, the service mesh can be disabled,... in this case, the service mesh will be injected as init container to lift the deny rule..."